resources/definitions/fdmprinter.def.json

```diff
@@ -154,6 +154,21 @@
                     "settable_per_extruder": false,
                     "settable_per_meshgroup": false
                 },
+                "machine_buildplate_type":
+                {
+                    "label": "Build Plate Material",
+                    "description": "The material of the build plate installed on the printer.",
+                    "default_value": "glass",
```

Comment:
this means the engine G-code will now always output build plate type "glass" as comment even though most 3rd party printers use another type of plate?

Reply:
Yes, that is right. It it is worthy, the default value can be changed.